core/dbt/parser/read_files.py

```diff
@@ -108,68 +117,44 @@ def get_source_files(project, paths, extension, parse_file_type, saved_files):
     return fb_list
 
 
-def read_files_for_parser(project, files, dirs, extension, parse_ft, saved_files):
-    parser_files = []
-    source_files = get_source_files(
-        project, dirs, extension, parse_ft, saved_files
-    )
-    for sf in source_files:
-        files[sf.file_id] = sf
-        parser_files.append(sf.file_id)
-    return parser_files
-
-
 # This needs to read files for multiple projects, so the 'files'
```

Comment:
I think this command can be removed. They are valid questions, but it does not help the user to understand the code better. If this is something of concern, we should create an issue for it.